src/harness/testcases/deregistration_testcase.py

```diff
@@ -77,4 +77,21 @@ def test_10_15_4_2_1(self):
     response = self._sas.Deregistration(request)['deregistrationResponse'][0]
     # Check the deregistration response
     self.assertFalse('cbsdId' in response)
-    self.assertEqual(response['response']['responseCode'], 102)
+    self.assertEqual(response['response']['responseCode'], 102)    
+
+  @winnforum_testcase
+  def test_WINFF_FT_S_DER_5(self):
+    """CBSD deregistration request when CBSD ID does not exist in SAS.
+
+    CBSD sends deregistration request when its CBSD ID is not in SAS,
+    the response should be FAIL.
+    """
+
+    # Deregister the device before registration, thus the CBSD ID does not exist in SAS
+    request = {'deregistrationRequest': [{'cbsdId': 'A nonexistent cbsd id'}]}
+    response = self._sas.Deregistration(request)['deregistrationResponse'][0]
+    # Check the deregistration response
+    self.assertFalse('cbsdId' in response)
+    self.assertEqual(response['response']['responseCode'], 103)
```

Comment:
Actually 105 doesn't make sense to me though for both test cases(DER.3 and DER.5). For DER.3, cbsdId is not even provided, so how could SAS return 105. For DER.5, cbsdId is nonexistent, so SAS should not return 105. Should we request test plan be modified?